src/identity/components/Socials.tsx

```diff
@@ -0,0 +1,117 @@
+import type { Address, Chain } from 'viem';
+import { githubSvg } from '../../internal/svg/githubSvg';
+import { twitterSvg } from '../../internal/svg/twitterSvg';
+import { warpcastSvg } from '../../internal/svg/warpcastSvg';
+import { websiteSvg } from '../../internal/svg/websiteSvg';
+import { border, cn, pressable } from '../../styles/theme';
+import { useName } from '../hooks/useName';
+import { useSocials } from '../hooks/useSocials';
+import { useIdentityContext } from './IdentityProvider';
+
+type SocialsReact = {
+  address?: Address | null;
+  ensName?: string;
+  chain?: Chain;
+  className?: string;
+};
+
+export function Socials({ address, chain, className }: SocialsReact) {
+  const { address: contextAddress, chain: contextChain } = useIdentityContext();
+
+  const accountAddress = address ?? contextAddress;
+  const accountChain = chain ?? contextChain;
+
+  if (!accountAddress) {
+    console.error(
+      'Socials: an Ethereum address must be provided to the Socials component.',
+    );
+    return null;
+  }
```

Comment:
is there history behind why Identity doesn't default to the connected address?

Reply:
Hmm I'm not sure what prompted the decision - something Leo implemented